geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -7,27 +7,138 @@ Version 2.6.0
-  diff: https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-c0556a9b77c45b7cefd76100f480643b430c8ecc080edc549cee740cfb71e6aa
-- All the configuration present in the controllers (especially the map configuration) is moved to the
-  `vars.yaml` file, section `vars/interfaces_config/<interface>`.
-  diff: https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-98196fd1bacd50a637a3fec9a1a4116cb14eca3587d11c76d734deaf02ae3296
-- All the configuration present in the *.html.ejs files is moved to the `vars.yaml` file,
-  section `vars/interfaces_config/<interface>`.
-  diff: https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-49e4e3e08323fa260d0e7682f102a585bb0ee2d25f0434e47255519c0b0423a5
-
-Changes in the `geoportal/vars.yaml` file:
-  https://github.com/camptocamp/demo_geomapfish/compare/prod-2-5...prod-2-6#diff-d8128a088c1fb76a5d2e77bfc707d1426f5bd1511d430f20a724c0e4955b3db7
-
-- Optional point: the header can be in a separate file, diff in the `*.html.ejs` file:
+Client application configuration
+................................
+
+Acctually we have some configuration in the `*.html.ejs`, in the `Controler*.js`, and in the vars.yaml file.
+The goal of this change is to move all thies configurationthe the `vars.yaml` file (with default values in
+the `CONST_vars.yaml`). This will make the future upgrade easyer and incrase the configuration possibility
+in the simple application mode, see the documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/create_application.html#simple-application>.
+
+
+When you apply the `ngeo.diff` you should first apply the changes in the `geoportal/vars.yaml` file.
+
+For the next you should consultate the documentation about:
+- The interfaces_config documentation
+<https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/ngeo.html?highlight=ngeo#dynamic-json-view>
+- GMF constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-contribs_gmf_src_options.html>
+- ngeo constants definitions <https://camptocamp.github.io/ngeo/master/jsdoc/module-src_options.html>
+
+The 'srid', 'alternate_projections', 'resolutions' and 'extent' `add path` will be configure on one point and will be
+dispatch using c2ctemplate or YAML link.
+
+Kind of thinks you can see:
+
+Example in API
+..............
+
+In the ngeo.diff:
+```
+--- a/geoportal/cartoriviera_geoportal/static-ngeo/api/index.js
++++ b/geoportal/cartoriviera_geoportal/static-ngeo/api/index.js
+...
+-// The URL to the search service.
+-config.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+And in your `geoportal/cartoriviera_geoportal/static-ngeo/api/index.js` file:
+```
+// The URL to the search service.
+onfig.searchUrl = '{FULL_ENTRY_POINT}search?interface=api&limit=15';
+```
+We see that that your settings didn't corresponds to the previus default settings, than you should have in
```

Comment:
We see that the project settings didn't correspond to the previous default settings, therefore you should have in
your vars file something like: